portfolio/src/main/webapp/script.js

```diff
@@ -54,3 +54,9 @@ window.onload = () => {
     .then(text => { document.getElementById("post1").innerHTML = text; 
   });
 }
+
+getServerMessage = () => {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('message-container').innerText = quote;
+  });
+}
```

Comment:
Comment all functions in code.

Any public functions in classes, any global-type fn's too.